Documentation/gettingstarted/servicemesh/envoy-custom-listener.rst

```diff
@@ -0,0 +1,62 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_envoy_custom_listener:
+
+*******************
+L7 Path Translation
+*******************
+
+This example replicates the Prometheus metrics listener which is
+already available via Cilium Agent ``--proxy-prometheus-port`` command
```

Comment:
I think removing "Cilium Agent" is fine, because it's implied (feel free to add it back if you disagree). A definite article is needed though. 
```suggestion
already available via the command line option ``--proxy-prometheus-port``.
```